dts/bindings/rng/zephyr,psa-crypto-rng.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
Is this copyright correct?

Reply:
This file was firstly created by @superna9999, so I assumed he set this properly.
Do you think I should change it?

Reply:
nah, keep it.